source/elements/oneDAL/source/common_interface/header_files_and_namespaces.rst

```diff
@@ -16,18 +16,38 @@ oneDAL public identifiers are represented in the following header files:
      - A header file for a particular algorithm.
        The folder for the algorithm itself is ``oneapi/dal/algo/%ALGO%/``.
 
-       The string %ALGO% should be substituted with the name of the algorithm, for example, ``kmeans`` or ``knn``.
+       The string %ALGO% should be substituted with the name of the algorithm,
+       for example, ``kmeans`` or ``knn``.
 
    * - ``oneapi/dal/algo/misc/%FUNC%.hpp``
-     - A header file for miscellaneous data types and functionality that is intended to be used by oneDAL algorithms and applications
+     - A header file for miscellaneous data types and functionality that is
+       intended to be used by oneDAL algorithms and applications
        of the analytical pipeline.
 
-       The string %FUNC% should be substituted with the functionality name, for example, ``mt19937`` or ``cross_enthropy_loss``.
+       The string %FUNC% should be substituted with the functionality name, for
+       example, ``mt19937`` or ``cross_enthropy_loss``.
+
+   * - ``oneapi/dal/table/%FILE%.hpp``
+     - A header file for types and entities related to the :txtref:`table` concept.
+
+       The string %FILE% should be substituted with the functionality name, for
+       example, ``common`` for base concepts or ``homogen`` for
+       ``homogen_table``. For entities that have ``_table`` suffix in names,
+       a related file should not contain this suffix in its name.
+
+   * - ``oneapi/dal/io/%FILE%.hpp``
+     - A header file for types and entities of input-output functionality.
```

Comment:
terminology: we introduced 'dataset', 'data source', etc, and here we use 'input-output'- need to define and set the connection between all of them